tools/disass/ida/disass.py

```diff
@@ -136,6 +139,87 @@ def get_direct_branch_target(branch_inst_ea):
         branch_inst_ea, target_ea))
     return target_ea
 
+# source: http://reverseengineering.stackexchange.com/a/1648/4197
```

Comment:
There is a subtlety to how `program.get_basic_block` is used that isn't explained but shouldn't be. It's really the `analyse_subroutine` and `analyse_block` that fills up the backing cached data structures, so before that happens, you'll get back empty things.

I think there is probably a way to dereference the data, assuming it's a pointer into something like the GOT/PLT. You may be able to inspect the address in IDA's `inst_t`. Another approach may be to look for data cross-references from the indirect call.

Each instruction could do that, and you can add it in. Why is that needed, though?